packages/components/src/components/tab-group/tab-group.stories.ts

```diff
@@ -0,0 +1,451 @@
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+const { argTypes, parameters } = storybookDefaults('sd-tab-group');
+const { overrideArgs } = storybookHelpers('sd-tab-group');
+const { generateTemplate } = storybookTemplate('sd-tab-group');
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+export default {
+  title: 'Components/sd-tab-group',
+  component: 'sd-tab-group',
+  args: overrideArgs({
+    type: 'slot',
+    name: 'default',
+    value: `
+    <sd-tab slot="nav" panel="tab-1">Tab 1</sd-tab>
+    <sd-tab slot="nav" panel="tab-2">Tab 2</sd-tab>
+    <sd-tab slot="nav" panel="tab-3" disabled>Tab 3</sd-tab>
+    <sd-tab slot="nav" panel="tab-4">Tab 4</sd-tab>
+    <sd-tab slot="nav" panel="tab-5">Tab 5</sd-tab>
+
+  
+    <sd-tab-panel name="tab-1"><div class="slot slot--text slot--border">Tab panel 1</div></sd-tab-panel>
+    <sd-tab-panel name="tab-2"><div class="slot slot--text slot--border">Tab panel 2</div></sd-tab-panel>
+    <sd-tab-panel name="tab-3"><div class="slot slot--text slot--border">Tab panel 3</div></sd-tab-panel>
+    <sd-tab-panel name="tab-4"><div class="slot slot--text slot--border">Tab panel 4</div></sd-tab-panel>
+    <sd-tab-panel name="tab-5"><div class="slot slot--text slot--border">Tab panel 5</div></sd-tab-panel>
+            `
+  }),
+  argTypes,
+  parameters: { ...parameters, docs: { story: { inline: false, height: '275px' } } },
+  decorators: [withActions] as any
+};
+
+/**
+ * Default: This shows sd-tab-group in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * Use the variant attribute to alternate between the `default` and `container` styles.
+ */
+
+export const Variant = {
+  parameters: { controls: { exclude: 'variant' }, docs: { story: { inline: false, height: '375px' } } },
+
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        x: { type: 'attribute', name: 'variant' }
+      },
+      args
+    });
+  }
+};
+
+/**
+ * The sd-tab-group becomes scrollable when there are more tabs than horizontal space allows.
+ */
+
+export const Scrollable = {
```

Comment:
could you either addd more panels or restrict the wrapper size? On my screen (4k display) its not scrollable until I resize the window which makes it a tiny little bit "unclear" what to do with it.
![image](https://github.com/solid-design-system/solid/assets/48211526/36b28104-231d-43b1-9de3-627a894543c1)